lib/rubygems/config_file.rb

```diff
@@ -457,9 +458,8 @@ def to_yaml # :nodoc:
 
   # Writes out this config file, replacing its source.
   def write
-    unless File.exist?(File.dirname(config_file_name))
-      FileUtils.mkdir_p File.dirname(config_file_name)
-    end
+    require 'fileutils'
+    FileUtils.mkdir_p File.dirname(config_file_name)
```

Comment:
[note] `FileUtils.mkdir_p` does nothing if the given path exists, so I remove `File.exist?`.